mgradm/shared/templates/attestationServiceTemplate.go

```diff
@@ -0,0 +1,62 @@
+// SPDX-FileCopyrightText: 2024 SUSE LLC
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package templates
+
+import (
+	"io"
+	"text/template"
+)
+
+const attestationServiceTemplate = `# uyuni-server-attestation.service, generated by mgradm
+# Use an uyuni-server-attestation.service.d/local.conf file to override
+
+[Unit]
+Description=Uyuni server attestation container service
+Wants=network.target
+After=network-online.target
+
+[Service]
+Environment=PODMAN_SYSTEMD_UNIT=%n
+Restart=on-failure
+ExecStartPre=/bin/rm -f %t/uyuni-server-attestation.pid %t/%n.ctr-id
+ExecStartPre=/usr/bin/podman rm --ignore --force -t 10 {{ .NamePrefix }}-server-attestation
+ExecStart=/usr/bin/podman run \
+	--conmon-pidfile %t/uyuni-server-attestation.pid \
+	--cidfile=%t/%n.ctr-id \
+	--cgroups=no-conmon \
+	--sdnotify=conmon \
+	-d \
+	-e database_connection  \
+	-e database_user \
+	-e database_password \
+	--replace \
+	--name {{ .NamePrefix }}-server-attestation \
+	--hostname {{ .NamePrefix }}-server-attestation \
```

Comment:
I think we should use the same internal domain as the server-image has.
```suggestion
	--hostname {{ .NamePrefix }}-server-attestation.mgr.internal \
```

Reply:
Fixed